app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ViewProfileScreenTest.kt

```diff
@@ -418,4 +425,59 @@ class ViewProfileScreenTest {
 
     verify(mockNavigationActions).navigateTo(Screen.PARKING_DETAILS)
   }
+
+  @Test
+  fun whenNoReviews_showsEmptyMessage() {
+    composeTestRule.waitForIdle()
+
+    // Scroll to the reviews section using swipe
+    composeTestRule.onNodeWithTag("ProfileContentSections").performTouchInput { swipeUp() }
+
+    composeTestRule.onNodeWithTag("UserReviewsTitle").assertIsDisplayed()
+
+    composeTestRule
+        .onNodeWithTag("NoReviewsMessage")
+        .assertIsDisplayed()
+        .assertTextEquals("You haven't written any reviews yet")
+  }
+
+  @Test
+  fun whenUserHasReviews_showsReviewsList() {
+    // Add test parkings to mock repository
+    mockParkingRepository.addParking(TestInstancesParking.parking1, {}, {}) // For review5
+    mockParkingRepository.addParking(TestInstancesParking.parking2, {}, {}) // For review1
+
+    // Add user1's reviews from TestInstancesReview
+    mockReviewRepository.addReview(TestInstancesReview.review1, {}, {})
+    mockReviewRepository.addReview(TestInstancesReview.review5, {}, {})
+
+    // Trigger review fetch for user1
+    reviewViewModel.getReviewsByOwnerId("user1")
+
+    composeTestRule.waitForIdle()
+
+    repeat(3) {
+      composeTestRule.onNodeWithTag("ProfileContentSections").performTouchInput { swipeUp() }
+    }
+
+    // Verify first review content (review1)
+    composeTestRule.onNodeWithTag("ParkingName_1").assertTextEquals("Rude épais")
+    composeTestRule.onNodeWithTag("RatingText_1").assertTextEquals("You rated this parking: 5.0 ⭐")
+    composeTestRule.onNodeWithTag("YouSaidText_1").assertIsDisplayed()
+    composeTestRule.onNodeWithText("\"Great parking!\"").assertIsDisplayed()
```

Comment:
try to keep your test tag as clear as possible. Someone who needs to modify your codes should be able to understand directly what the test tag is tagging

Reply:
which one is unclear ?

Reply:
Well `Great parking` for example is not really clear. Plus if I can picky, it is not coherent with the formatting of the other test tag.

Reply:
On the other hand this test is really good and very complete so good job!